server/plugin.go

```diff
@@ -923,3 +971,21 @@ func (p *Plugin) OnSharedChannelsSyncMsg(msg *model.SyncMsg, _ *model.RemoteClus
 
 	return resp, nil
 }
+
+func (p *Plugin) ChannelHasBeenCreated(c *plugin.Context, channel *model.Channel) {
```

Comment:
Given we're still relying on hooks to make shared channels work (for now), I'm guessing that in the event of a downtime incident, we'll want to plan to rotate the plugin to trigger the bulk share channel calls on startup?

Reply:
That is exactly what we are doing in the migration that we created.

Reply:
Right, I'm referring to steps we would need to manually take if an incident occurred, specifically that we'd want to re-run that migration by ensuring we rotate a pod and get the plugin to start from scratch?

Reply:
Every restart of the plugin, including configuration changes, are going to try to run the migration again.